allauth/account/tests/test_reset_password.py

```diff
@@ -280,6 +280,17 @@ def test_password_reset_ACCOUNT_LOGIN_ON_PASSWORD_RESET(self):
         # EmailVerificationMethod.MANDATORY sends us to the confirm-email page
         self.assertRedirects(resp, "/confirm-email/")
 
+    @override_settings(ACCOUNT_LOGOUT_ON_PASSWORD_CHANGE=True)
+    def test_password_change_ACCOUNT_LOGOUT_ON_PASSWORD_CHANGE(self):
```

Comment:
We're in the process of moving to pytest, and therefore adding new old style tests is strongly discouraged. Can you turn this into a pytest? Just to get you started:

``` python
def test_passowrd_change_account_logout_on_password_change(settings, auth_client):
    settings.ACCOUNT_LOGOUT_ON_PASSWORD_CHANGE = True
    resp = auth_client.post(...)
    assert resp['location'] == ...